app/js/reducers/addDrugOrderReducer.js

```diff
@@ -14,11 +18,19 @@ const addDrugOrderReducer = (state = initialState, action) => {
       return {
         ...state,
         addedOrder: action.data,
+        status: {
+          error: false,
+          added: true,
+        },
       };
     case POST_DRUG_ORDER_FAILURE:
       return {
         ...state,
-        error: action.payload,
+        errorMessage: action.payload.response.data.error.message,
```

Comment:
maybe a liitle bit of destructing would be nice here to avoid long chains like this, what do you think @Efosaok

Reply:
I am accessing this data in just one instance here, therefore, I see destructuring not really useful here.   😄

Reply:
Ok cool